samples/src/main/java/com/google/cloud/examples/speech/v1/SpeechTranscribeEnhancedModel.java

```diff
@@ -70,7 +70,6 @@ public static void sampleRecognize(String localFilePath) {
     try (SpeechClient speechClient = SpeechClient.create()) {
 
       // The enhanced model to use, e.g. phone_call
-      // Currently phone_call is the only model available as an enhanced model.
```

Comment:
This is generated by samplegen

Reply:
Noted. It's inaccurate at this point though. Presumably changing one comment won't break anything, since we don't intend to continue generating samples the same way?